### DISCUSSION TITLE
<< APIs from other services >>

### DISCUSSION THREAD

**Discussion by icamps** [texstudio-org/texstudio#3898] [Q&A]
Hello, 

I would like to know how can I add APIs from other models/systems I already have access.

For example, I have API from the 1minAI app ([link](https://gleaming-wren-2af.notion.site/1min-AI-API-Docs-111af080bd8f8046a4e6e1053c95e047)) that I would like to use with TeXstudio.

Regards,

Camps

**Comment by sunderme** [comment]
txs supports the chatgpt api which is used by many ai providers.
Adapting to other APIs means coding it in cpp.
